### ISSUE TITLE
Form - Follow best practices for displaying server-side form errors

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1252]
CENG_3323

Docs: https://react-magma.cengage.com/version/4.3.0/api/form/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/error-identification

The form shows a static form error positioned above the first form element, so it cannot be tested for accessibility.

Follow best practices for displaying server-side form errors-- see the advice ticket for complete details.  Some important points to keep in mind:

- Mark the container for the errors with role="alert", which will cause screen readers to read the errors upon display.
- Show the errors in a list.
- Use JavaScript to set focus to the top of the error list when it is displayed.
